content/millennium/r4/clinical/care-provision/care-team.md

```diff
@@ -56,7 +56,7 @@ _Implementation Notes_
  `encounter`  | Required if `_id` or `patient` is not present       | [`reference`] | Who care team is for. Example: `encounter=98765`
  `category`   | N                                                   | [`token`]     | The scope of care team being searched for. Examples: `category=longitudinal`
  `status`     | N                                                   | [`token`]     | Indicates the status of the care team
-`_revinclude`      | No                                             | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
+`_revinclude` | N                                                   | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
```

Comment:
let's put the example in a code block to match the ones above

Reply:
Yep
![image](https://github.com/cerner/fhir.cerner.com/assets/144141792/b2481101-1fd8-4d05-9fcc-80c7f6d86b43)